packages/web/pages/user/my-account/index.js

```diff
@@ -107,14 +118,137 @@ const InnerForm = ({ form, user, message, loading }) => {
 						validation={{ required: true }}
 					/>
 				</Cell>
-				<Cell>
+				<Cell col={3}>
+					<InputField
+						form={form}
+						name="secondary_email"
+						type="email"
+						label={t('account:labels.alternativeEmail')}
+						defaultValue={user.secondary_email || ''}
```

Comment:
```suggestion
						defaultValue={user?.secondary_email ?? ''}
```

Reply:
https://lea.verou.me/2020/06/refactoring-optional-chaining-into-a-large-codebase-lessons-learned/

Reply:
Não sabia: "Contrary to the logical OR (||) operator, the left operand is returned if it is a falsy value which is not null or undefined. In other words, if you use || to provide some default value to another variable foo, you may encounter unexpected behaviors if you consider some falsy values as usable (eg. '' or 0). See below for more examples."

:+1: